css/properties/padding-bottom.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "padding-bottom": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/padding-bottom",
+          "support": {
+            "webview_android": {
+              "version_added": null
```

Comment:
true 
padding-* are such basic properties, that all supports it (but eventually Opera Mini but we don't track it)